vagrant/Vagrantfile

```diff
@@ -93,6 +103,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     v.memory = 1024
     v.cpus = 1
     override.vm.box = box_name
+    if override_box_url
+      override.vm.box_url = override_box_url
+    end
```

Comment:
Should this box_url override live under the libvirt config? I got a libvirt box with --provider=virtualbox

```
https://ci.centos.org/artifacts/sig-atomic/centos-continuous/images-alpha/cloud/latest/images/centos-atomic-host-7-vagrant-libvirt.box
```